src/pages/SelectBallotPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import ReactSVG from 'react-svg'
+import styled from 'styled-components'
+
+import Main, { MainChild } from '../components/Main'
+import Prose from '../components/Prose'
+
+const Legend = styled.fieldset`
+  margin: auto;
+`
+const CodeBox = styled.div`
+  max-width: 320px;
+  margin: auto;
+`
+
+let resetActivationCode: number
+
+const StartPage = (props: RouteComponentProps) => {
+  const [activationCode, setActivationCode] = useState('')
+  const goToStartPage = () => {
+    props.history.push('/start')
+  }
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setActivationCode(event.target.value)
+    if (event.target.value === 'MyVoiceIsMyPassword') {
+      clearTimeout(resetActivationCode)
+      goToStartPage()
+    } else {
+      clearTimeout(resetActivationCode)
+      resetActivationCode = window.setTimeout(() => {
+        setActivationCode('')
+      }, 1000)
+    }
+  }
+  const onBlur = (event: React.ChangeEvent<HTMLInputElement>) => {
+    event.target.focus()
+  }
+
+  return (
+    <Main>
+      <MainChild center>
+        <fieldset>
+          <Legend>
+            <label htmlFor="ActivationCode">
+              <Prose textCenter>
+                <h1>Load Your Ballot</h1>
```

Comment:
change to `Scan Your Activation Code`. I don't think voters are going to think of their QR code as their ballot and it's going to be confusing to them, even if it includes the ballot style.

Reply:
Totes. All language needs to be reviewed.

Reply:
Fixed.